### PR TITLE
update .gitignore

### PR DESCRIPTION
**Pull request by AlwinEsch**
Also comes the next time this to the others.

Currently I check about the build problem and hope to bring the fix.

### CODE REVIEW

**Comment by garbear** [comment]
Can I give you merge permissions for the kodi-game org? it's easier to push directly than open 60 PRs

**Comment by AlwinEsch** [comment]
Good idea, can prevent much time :)

**Comment by fetzerch** [comment]
@garbear: wouldn't it be simpler if i just add that to kodi-game-scripting?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,33 @@
...
+# CLion
+/.idea
+
+# clion
+.idea/
+
+# KDev
+*.kdev4
+
+*.patch
```
Comment (garbear) [inline .gitignore]: Remove these two lines, .patch files placed in depends/common/2048 are applied to the 2048 source

**Comment by fetzerch** [comment]
Applied to all cores.

**Pull request closed without merging**